src/lib/import-helper.js

```diff
@@ -0,0 +1,631 @@
+/*
```

Comment:
just to be 100% sure this is just a rename from the old import ? or is it modified too ?

Reply:
It's just a rename, no real changes were made. A lot of the functions in here are only used by `import.js`, hence the naming :thumbsup:

Reply:
Not sure why we are just moving code around like this ... what is the benefit here?

Reply:
Yeah it is just moving code but this makes to easier to find exactly what you need to import the console config

The other file contains a lot of file processing funcs which really are just helpers